### PR TITLE
Brainbrowser: Added Spec/Readme

### PR DESCRIPTION
**Pull request by MounaSafiHarab**
Spec for Brainbrowser

### CODE REVIEW

File: modules/brainbrowser/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+slices. Multiple volumes can be displayed simultaneously and coordinates
+synchronized across the multiple volumes, if desired. Image brightness and 
+intensity can be set by the user.
+A `Play` button is also available to make feasible the navigation through the 
+time dimension of a 4-Dimensional image (such as resting state fMRI scan). 
+A measurement tool is also available allowing radiologist to report measurements
+of an incidental finding.
+
+
+## Permissions
```
Comment (MounaSafiHarab) [inline modules/brainbrowser/README.md]: @driusan 

I know we can type the url directtly to access brainbrowser, but a user should theoretically go to it through imaging browser which is why I thought I would link to that section in Imaging Browser. Do you think it is correct to do it this way?

**Comment by MounaSafiHarab** [comment]
@cmadjar 
if you have a minute, please take a look at this readme; see if you disagree with the text!

File: modules/brainbrowser/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+
+## Scope
+
+Brainbrowser allows navigating volumes in sagittal, coronal, and transverse 
+slices. Multiple volumes can be displayed simultaneously and coordinates
+synchronized across the multiple volumes, if desired. Image brightness and 
+intensity can be set by the user.
+A `Play` button is also available to make feasible the navigation through the 
+time dimension of a 4-Dimensional image (such as resting state fMRI scan).
```
Comment (cmadjar) [inline modules/brainbrowser/README.md]: I rewrote a little bit the section above moving around sentences:

"Brainbrowser allows visualization of MINC images directly within the browser. The user can navigate through those images across their sagittal, coronal and transverse axes. In addition, a `Play` button is also available for 4-Dimensional datasets to allow navigation through their fourth dimension (such as the time dimension for resting state fMRI scans). 
Multiple volumes can be displayed simultaneously with the possibility to synchronize all volumes' navigation when moving the cursor across one of the volume"

File: modules/brainbrowser/README.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+intensity can be set by the user.
+A `Play` button is also available to make feasible the navigation through the 
+time dimension of a 4-Dimensional image (such as resting state fMRI scan). 
+A measurement tool is also available allowing radiologist to report measurements
+of an incidental finding.
+
+
+## Permissions
+
+Since Brainbrowser is accessible from Imaging Browser, its permissions are
```
Comment (cmadjar) [inline modules/brainbrowser/README.md]: replace "from Imaging Browser" by "from the Imaging Browser module"

File: modules/brainbrowser/README.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+A `Play` button is also available to make feasible the navigation through the 
+time dimension of a 4-Dimensional image (such as resting state fMRI scan). 
+A measurement tool is also available allowing radiologist to report measurements
+of an incidental finding.
+
+
+## Permissions
+
+Since Brainbrowser is accessible from Imaging Browser, its permissions are 
+directly linked to those of Imaging Browser. Please consult the Imaging Browser
```
Comment (cmadjar) [inline modules/brainbrowser/README.md]: replace "linked to those of Imaging Browser." by "related to those of the Imaging Browser module."

File: modules/brainbrowser/README.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+## Scope
+
+Brainbrowser allows navigating volumes in sagittal, coronal, and transverse 
+slices. Multiple volumes can be displayed simultaneously and coordinates
+synchronized across the multiple volumes, if desired. Image brightness and 
+intensity can be set by the user.
+A `Play` button is also available to make feasible the navigation through the 
+time dimension of a 4-Dimensional image (such as resting state fMRI scan). 
+A measurement tool is also available allowing radiologist to report measurements
```
[the remark this answers is not in the document]
Comment (MounaSafiHarab) [reply modules/brainbrowser/README.md]: yes, I think it should have an 's' (although I am not an anglophone!)

File: modules/brainbrowser/README.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+## Purpose
+
+Brainbrowser is intended to allow users to visualize candidate scan session MINC 
+volumes collected for a study.
+
+## Intended Users
+
+The two primary types of users are:
+- Imaging specialists doing online QC
```
[the remark this answers is not in the document]
Comment (MounaSafiHarab) [reply modules/brainbrowser/README.md]: ok

File: modules/brainbrowser/README.md (hunk 3 above)
Comment (MounaSafiHarab) [reply modules/brainbrowser/README.md]: done

File: modules/brainbrowser/README.md (hunk 4 above)
Comment (MounaSafiHarab) [reply modules/brainbrowser/README.md]: done

File: modules/brainbrowser/README.md (hunk 2 above)
Comment (MounaSafiHarab) [reply modules/brainbrowser/README.md]: I will use mostly what you suggested with some modifications (for example axes is in fact slices, ...).
Check the latest commit and let me know!

File: modules/brainbrowser/README.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
+# Brainbrowser
+
+## Purpose
+
+Brainbrowser is intended to allow users to visualize candidate scan session MINC 
+volumes collected for a study.
```
[the remark this answers is not in the document]
Comment (driusan) [reply modules/brainbrowser/README.md]: isn't it also linked to from the mri violations module and doesn't it also allow visualization of nifti?

File: modules/brainbrowser/README.md (hunk 2 above)
Comment (driusan) [reply modules/brainbrowser/README.md]: maybe also mention DTI for a non-temporal 4th dimension example.

File: modules/brainbrowser/README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (driusan) [reply modules/brainbrowser/README.md]: yes, there should be an s (and no apostrophe)

File: modules/brainbrowser/README.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+time dimension of a 4-Dimensional image (such as resting state fMRI scan). 
+A measurement tool is also available allowing radiologist to report measurements
+of an incidental finding.
+
+
+## Permissions
+
+Since Brainbrowser is accessible from Imaging Browser, its permissions are 
+directly linked to those of Imaging Browser. Please consult the Imaging Browser 
+[Permissions](https://github.com/aces/Loris/blob/minor/modules/imaging_browser/README.md#imaging_browser_perm_link)
```
Comment (driusan) [inline modules/brainbrowser/README.md]: The link shouldn't be to GitHub, it should be relative to the location of the README being read.

Comment (MounaSafiHarab) [reply modules/brainbrowser/README.md]: in this case, I am pointing to the README of imaging browser (another module); which is modified in this pull request as well, so I would start the link from the modules/ directory?

File: modules/brainbrowser/README.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (MounaSafiHarab) [reply modules/brainbrowser/README.md]: Talked to Dave about this, and he is right, if we want to say Imaging browser, we have to say violated scans as well; and we usually do not track where the module is accessible from (but rather, the other way around, so Cecile I will remove the imaging browser statement in the coming commit)!

File: modules/brainbrowser/README.md (hunk 8 above)
Comment (MounaSafiHarab) [reply modules/brainbrowser/README.md]: Okay, so I will use `../imaging_browser/` ...
thanks!

File: modules/brainbrowser/README.md [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+- Project radiologists reporting incidental findings.
+
+
+## Scope
+
+Brainbrowser allows visualization of MINC images directly within its browser.
+The user can navigate those images with sagittal, coronal, and transverse 
+slices displayed.  In addition, a `Play` button is available to make feasible 
+the navigation through a 4-Dimensional image (such as time in the case of 
+resting state fMRI scans, and directions in the case of DWI scans).
```
Comment (MounaSafiHarab) [inline modules/brainbrowser/README.md]: @cmadjar 
it is called directions for DWI? please confirm!

File: modules/brainbrowser/README.md [hunk 10]
```diff
@@ -17,9 +17,9 @@ The two primary types of users are:
 
 Brainbrowser allows visualization of MINC images directly within its browser.
 The user can navigate those images with sagittal, coronal, and transverse 
-slices displayed.  In addition, a `Play` button is also available to make 
-feasible the navigation through the time dimension of a 4-Dimensional image 
-(such as resting state fMRI scans). 
+slices displayed.  In addition, a `Play` button is available to make feasible 
+the navigation through a 4-Dimensional image (such as time in the case of 
+resting state fMRI scans, and directrions in the case of DWI scans).
```
Comment (driusan) [inline modules/brainbrowser/README.md]: Are directrions a new type of subatomic particle?

Comment (driusan) [reply modules/brainbrowser/README.md]: "DWI scans work by measuring the radioactive decay of directrions"

File: modules/brainbrowser/README.md (hunk 9 above)
Comment (cmadjar) [reply modules/brainbrowser/README.md]: yes :)

**Pull request merged**